tests/queries/0_stateless/01109_exchange_tables.sql

```diff
@@ -13,17 +13,23 @@ CREATE TABLE t2 ENGINE=MergeTree() ORDER BY tuple() AS SELECT rowNumberInAllBloc
 EXCHANGE TABLES t1 AND t3; -- { serverError 60 }
 EXCHANGE TABLES t4 AND t2; -- { serverError 60 }
 RENAME TABLE t0 TO t1; -- { serverError 57 }
-DROP TABLE t1;
-RENAME TABLE t0 TO t1;
+
```

Comment:
It's better to make new test for these queries. Or, at least, add new queries to the end of this test without modifying existing queries.

Reply:
ACK, has fixed.